src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -50,3 +50,30 @@ def test_get_self_cal_last_date_and_time():
 
 def test_self_cal(session):
         session.self_cal()
+
+
+def test_standard_waveform(session):
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_FUNC
+    session.configure_standard_waveform(nifgen.Waveform.SINE, 2.0, 0.0, 0.0, 2000000)
+    session.output_enabled = True
```

Comment:
I don't see the need to set attributes in lines 58, 59, 60 in this test.
I don't see the need to initiate.
Keep the tests as granular as possible. Do as little as you can.